src/Collector.ts

```
@@ -325,14 +346,43 @@ export default class Collector {
   _referenceForSymbol(symbol:typescript.Symbol):types.ReferenceNode {
     this._walkSymbol(symbol);
     const referenced = this.types[this._nameForSymbol(symbol)];
-    if (referenced && referenced.type === 'interface') {
+    if (referenced && referenced.type === types.NodeType.INTERFACE) {
       referenced.concrete = true;
     }
 
     return {
-      type: 'reference',
+      type: types.NodeType.REFERENCE,
       target: this._nameForSymbol(symbol),
     };
   }
 
+  _directiveFromDocTag(jsDocTag:doctrine.Tag):types.DirectiveNode {
+    // This regex properly catches the parameters substring
+    const paramsContentRegex = /\s*\(\s*((?:.|\s)*)\s*\)\s*/g;
```

I was able to add `@someDirective (minStock: 1000) (test: 200)` without errors. `(test: 200)` was ignored though!
How about throwing an error if there are more than one `(...)`?

---

Nice! Actually the behavior is that anything put after the parenthesis is ignored. It's probably better to throw an error.